.pipelines/templates/.builder-release-template.yaml

```diff
@@ -170,6 +170,19 @@ steps:
         IS_NOT_1804=$( [[ "${OS_VERSION}" != "18.04" ]] && echo true || echo false ) && \
         echo "##vso[task.setvariable variable=IS_NOT_1804]${IS_NOT_1804}"
     displayName: 'Check if OS version is not 18.04'
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: agentbaker-vuln-to-kusto
+      scriptType: 'bash'
+      scriptLocation: 'inlineScript'
+      inlineScript: |
+        export AZCOPY_AUTO_LOGIN_TYPE=AZCLI
+        azcopy copy https://vulntokustobinary.blob.core.windows.net/vuln-to-kusto/v0.0.1-ee36fabc1ec4f415a022042ea89f751bf05da6ef/vuln-to-kusto-vhd_Linux_x86_64 ./vuln-to-kusto
```

Comment:
assuming this storage is locked down? does that mean we're already able to pull from it using the agent identity?

Reply:
yep the storage account is private and configured to be accessible with the identity used in this step (`agentbaker-vuln-to-kusto`)